ground/gcs/src/plugins/config/configstabilizationwidget.cpp

```diff
@@ -221,3 +262,57 @@ void ConfigStabilizationWidget::applyMWRateConvertDialog()
     }
 }
 
+void ConfigStabilizationWidget::showExpoPlot()
```

Comment:
Can this be factored any cleaner to reduce the amount of code duplication?  It's nice to minimize the amount plotted, but not absolutely essential.  Perhaps set flags for what is dirty in these conditionals and then if (horizonRollDirty) { m_stabilization->horizonStickExpoPlot->plotDataRoll...

Also, formatting and operator spacing.